
try{
    window.alert('hello')
   throw 'some thing went wrong'

}
catch(err){
    console.log(err)
}


